data-reporting-service/investigation-service/docker-compose.Dockerfile

```diff
@@ -1,14 +1,15 @@
 FROM amazoncorretto:17 as builder
 #Copy sources
-COPY . /usr/src/data-reporting-service/investigation-service
+COPY common-util /usr/src/data-reporting-service/common-util
+COPY investigation-service /usr/src/data-reporting-service/investigation-service
 COPY gradle /usr/src/data-reporting-service/gradle
 COPY gradlew /usr/src/data-reporting-service/gradlew
 
 #cd to investigation-service
```

Comment:
We are building the other DockerFile in root directory but this in the service directory? what's the difference?

This is for all the services

Reply:
Yeah this is only for local docker compose for local builds.